test/cppsuite/src/storage/scoped_cursor.cpp

```diff
@@ -26,7 +26,7 @@
  * OTHER DEALINGS IN THE SOFTWARE.
  */
 
-#include "scoped_types.h"
+#include "scoped_cursor.h"
```

Comment:
See 9665fac0dc89fe587edd294446bcb3c340235784

Reply:
I don't like this naming change as it results in wiredtigers codebase having two session.hs and two cursor.hs. I think we could make them .hpp? Ideally one day the cppsuite would be in its own repo but for now I would like some kind of differentiation.

Reply:
Well overlapping filenames can be an issue in a few scenarios but primarily for convenience, I feel like WiredTiger API headers should be unique so they can be easily navigated to. Additionally it could be confusing for new users to the codebase, and if any documentation references filenames (which I hope it doesn't) it could present an issue.

Reply:
We have decided to change `cursor.h/cpp` to `wiredtiger_cursor.h/cpp`, and follow the same pattern for `session.h/cpp`.

Reply:
Updated to `wiredtiger_cursor` in 7ec7c1acf9e1117a50634cbaeb01c2acec145858